includes/Rest/Site.php

```diff
@@ -11,7 +11,7 @@ class Site extends RestController {
     public function register_routes() {
         $this->post( '/settings', 'save_settings', 'can_manage_settings' );
         $this->post( '/settings/save-options', 'save_options', 'manage_options' );
-        $this->delete( '', 'delete_site' );
+        $this->delete( '', 'delete_site', 'save_options' );
```

Comment:
Cloud you please check `save_options` permission callback ones again? I don't see any `save_options` method here in that class.